tests/drivers/uart/uart_mix_fifo_poll/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/ {
+	chosen {
+		zephyr,console = &uart1;
+	};
+};
+
+&uart1 {
+	current-speed = <115200>;
+	status = "okay";
+	tx-pin = <6>;
+	rx-pin = <8>;
+	rts-pin = <0>;
```

Comment:
Is this right?  Deleting the property is supposed to be the way to mark that it's not supported.  This would seem to assign the XL1 pin to be both RTS and CTS.

By inspection using `-1` as the pin ordinal should also have the desired effect.  `0xffffffff` (the value of `NRF_UART_PSEL_DISCONNECTED` should also.

Reply:
yeah, it's a copy paste from test `uart_async_api`. Will clean it up.